Mixpanel/MixpanelPeople.h

```diff
@@ -89,6 +89,18 @@ NS_ASSUME_NONNULL_BEGIN
  @method
  
  @abstract
+ Unregister a specific device token from the ability to receive push notifications.
+ 
+ @discussion
+ This will remove the provided push token saved to this people profile. This is useful
+ in conjunction with a call to `reset`, or when a user is logging out.
+ */
+- (void)removeSinglePushDeviceToken:(NSData *)deviceToken;
```

Comment:
I like the idea of renaming the old method, but did not want to deprecate its functionality in case people are already using it in some specific manner. Open to that though